python/tvm/relay/frontend/tflite.py

```diff
@@ -384,7 +401,16 @@ def convert_logistic(self, op):
         input_tensor = input_tensors[0]
         in_expr = self.get_expr(input_tensor.tensor_idx)
 
+        output_tensors = self.get_output_tensors(op)
+        assert len(output_tensors) == 1, "output tensors length should be 1"
+        output_tensor = output_tensors[0]
+
+        if input_tensor.qnn_params:
+            in_expr = self.dequantize(in_expr, input_tensor)
         out = _op.sigmoid(in_expr)
```

Comment:
Does the INT computation of sigmoid is very complex so that we could only `dequantize->float compute->requantize`?

Reply:
Well I have the following understanding for that case: since by definition sigmoid outputs a probability in [0, 1] in real space, then in uint8 [0, 1] will have a different meaning, so we should dequantize before doing the operation. Correct me if I'm wrong.
Or you are talking about INT, not UINT?

Reply:
I was talking about **pure integer computation of sigmoid.** That is to say, current computation of quantized sigmoid is float arithmetic in fact, we just `dequantize` to float, `float arithmetic`, `requantize` to quantized UINT8 type. However, like TFLite, we have **pure integer computation of sigmoid.** like this : https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/internal/reference/integer_ops/logistic.h#L24, which is I checked just now. It seems a little complicated. So maybe we use current way is an acceptable way.

Reply:
If there is a need to improve quantized logistic performance, introducing a qnn.logistic sounds reasonable and then lowering it as part of the legalize_qnn pass would be what I would do: However that in my mind is the next step in terms of improving the performance for logistic , the first step on the other hand to make this work and the dequantize , operation, quantize is a good way of making progress.

Reply:
I had exact similar thoughts. As @FrozenGene pointed, the implementation for integer-only softmax looked complicated, so I decided to just support functionally for now.